hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/types/BucketMetadata.java

```diff
@@ -0,0 +1,264 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * <p>
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.ozone.recon.api.types;
+
+import com.google.common.base.Preconditions;
+
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlElement;
+import java.util.List;
+
+/**
+ * Metadata object represents one bucket in the object store.
+ */
+@XmlAccessorType(XmlAccessType.FIELD)
+public final class BucketMetadata {
```

Comment:
@ivandika3 Was it not possible to use existing OmBucketInfo or BucketObjectDBInfo (already in Recon) object ?

Reply:
Ditto.

Reply:
I have removed `BucketMetaata` and uses `BucketObjectDBInfo` instead.